lib/internal/Magento/Framework/Console/Cli.php

```diff
@@ -151,6 +178,11 @@ protected function getApplicationCommands()
     {
         $commands = [];
         try {
+            if (class_exists(\Magento\Setup\Console\CommandList::class)) {
+                $setupCommandList = new \Magento\Setup\Console\CommandList($this->serviceManager);
+                $commands = array_merge($commands, $setupCommandList->getCommands());
+            }
```

Comment:
Could you please explain, why this change is needed?

Reply:
> We don't need these lines; it is probably due to a merge issue. You can
follow the logic. These lines don't change logic, just were touched by the
commit

These lines were added, so let's remove them if we don't need them